src/contract/Contract.ts

```diff
@@ -81,7 +81,7 @@ interface Aci extends BaseAci {
   externalEncodedAci: any[];
 }
 
-interface Event {
+export interface Event {
```

Comment:
actually, `Event` is here because of https://github.com/aeternity/aepp-sdk-js/issues/1537, it is already defined in autorest-generated api